marge/approvals.py

```diff
@@ -8,16 +13,128 @@ class Approvals(gitlab.Resource):
 
     def refetch_info(self):
         gitlab_version = self._api.version()
-        if gitlab_version.release >= (9, 2, 2):
-            approver_url = '/projects/{0.project_id}/merge_requests/{0.iid}/approvals'.format(self)
-        else:
-            # GitLab botched the v4 api before 9.2.3
-            approver_url = '/projects/{0.project_id}/merge_requests/{0.id}/approvals'.format(self)
 
         if gitlab_version.is_ee:
-            self._info = self._api.call(GET(approver_url))
+            self._info = self._api.call(GET(self.get_approvals_url(self)))
         else:
-            self._info = dict(self._info, approvals_left=0, approved_by=[])
+            self._info = self.get_approvers_ce()
+
+    def get_approvers_ce(self):
+        """get approvers status using thumbs on merge request
+        """
+        gitlab_version = self._api.version()
+
+        # Gitlab supports approvals in free version after 13.2.0
```

Comment:
Does it have everything the same as in EE? If so you could move this out of `get_approvers_ce` and add it after `if gitlab_version.is_ee`. Otherwise it could be moved to right before `for approver in approval_api_results['approved_by']:`

Reply:
Yes, CE will return approvals from the `/approvals` endpoint as well.

Since it's been almost a year since the original PR, my question is if this version check should even be here at this point? If people are still running GitLab <= 13.1 they have [much bigger issues](https://www.cvedetails.com/vulnerability-list/vendor_id-13074/Gitlab.html) to worry about than marge-bot not being supported :grinning: